graphics/timg/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github  1.0
+
+github.setup        hzeller timg 1.6.0 v
+github.tarball_from archive
+revision            0
+categories          graphics
+
+maintainers         {outoforder.cc:urkle @urkle} openmaintainer
+
+description         Terminal Image and Video Viewer
+long_description    A user-friendly terminal image viewer that uses graphic \
+                    capabilities of terminals (Sixel, Kitty or iterm2), or 24-Bit \
+                    color capabilities and unicode character blocks if these are not \
+                    available.
+
+license             GPL-2
+homepage            https://timg.sh/
+
+
+checksums           rmd160  fc1b39f4395f7a4dc4da77f12e558289e05f3425 \
+                    sha256  9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af \
+                    size    1166246
+
+installs_libs       no
+
+depends_build-append \
+                    path:bin/pkg-config:pkgconfig
+
+depends_lib-append  port:GraphicsMagick \
+                    port:webp \
+                    path:include/turbojpeg.h:libjpeg-turbo \
+                    port:libexif \
+                    path:lib/pkgconfig/librsvg-2.0.pc:librsvg \
+                    path:lib/pkgconfig/cairo.pc:cairo \
+                    port:ffmpeg \
```

Comment:
`ffmpeg` is actually an old version (i.e., 4.x) so you likely want to use the recent `ffmpeg7` port. You'll need to tell it where to find the `.pc` files as with:

```
configure.env-append \
                    PKG_CONFIG_PATH=${prefix}/libexec/ffmpeg7/lib/pkgconfig:env(PKG_CONFIG_PATH)
```

Reply:
I ended up using configure.pkg_config_path-append instead as that seems to be the more correct way to define this.

Reply:
there is a reason I had it like this ;) Please install the `ffmpeg` or `ffmpeg6` ports and then install `timg` - how do you make sure it will pick up the libraries/headers from `ffmpeg7`? I haven't tested if to make 100% sure, but I think it will just pick whatever it finds first.

Reply:
I did test it. And verified that it used the correct one.

(without the pkg_config_path or your line it failed to compile completely with only ffmpeg7 installed)

and with both ffmpeg and ffmpeg7 installed using the pkg_config_path line it correctly picked ffmpeg7.

I guess, the only time it would be an issue is if someone explicitly modified their PKG_CONFIG_PATH env locally to include the macports stock path (/opt/local/lib/pkgconfig).

Though even doing that it still worked??

(in `sudo sh`)
```
export PKG_CONFIG_PATH=/opt/local/lib/pkgconfig/
port clean
port build
otool  -L work/build/src/timg
```